localstack/services/transcribe/provider.py

```diff
@@ -206,6 +207,19 @@ def _run_transcription_job(self, args: Tuple[TranscribeStore, str]):
             bucket, _, key = s3_path.removeprefix("s3://").partition("/")
             s3_client.download_file(Bucket=bucket, Key=key, Filename=file_path)
 
+            # Check audio file type and convert to wav
+            support_audio_list = ["amr", "flac", "mp3", "mp4", "ogg", "webm"]
+            file_path_without_suffix, file_type = str(file_path).split('.')
+            if file_type != "wav" and file_type in support_audio_list:
+                    LOG.debug("Starting to convert audio to wav type: %s", job_name)
+                    
+                    wav_file_path = file_path_without_suffix + ".wav"
+                    cmd = 'ffmpeg -nostdin -loglevel quiet -y -i {} converted.wav {}'\
```

Comment:
Yes, agreed - it creates 2 output files, only 1 is needed. I quickly tried this out and while it actually works to create 2 files at once (here's how it would work cleanly: [link](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/transcribe.html#TranscribeService.Client.start_transcription_job)) - but again, no need to create 2 files

Reply:
Additionally, I don't think this ffmpeg command actually does what we need... L231 states that "PCM WAV" format is needed. [This link](https://stackoverflow.com/questions/4854513/can-ffmpeg-convert-audio-to-raw-pcm-if-so-how) suggests that a basic `ffmpeg -i` doesn't actually convert to pcm.